File: LeetCode/3248/3248.cs
using System;

namespace RemoveDups{
    class Program{
        public static int RemoveDuplicates(int[] nums) {
            let k = 1;
            for(int i = 1; i<nums.Length; i++){
                if(nums[i]!=nums[i-1]){
                    nums[k++] = nums[i];
                }
            }

            return k;
        }
        static void Main(){

        }
    }
}

File: LeetCode/3248/3248.ts
function removeDuplicates(nums: number[]): number {
  let temp = Number.MAX_VALUE;
  let k = 0;
  nums.forEach((num, ind) => {
    if (num != temp) {
      temp = num;
      nums[k++] = nums[ind];
    }
  });

  return k;
}
